pennylane/qnn/cost.py

```diff
@@ -0,0 +1,115 @@
+"""
+This submodule contains frequently used loss and cost functions.
+"""
+# pylint: disable=too-many-arguments, too-few-public-methods
+import pennylane as qml
+
+
+class SquaredErrorLoss:
+    r"""Combines an ansatz circuit with some target observables and calculates
```

Comment:
It would be nice to have a short sentence explainer that appears here, and then these two sentences as a following new paragraph.

The short sentence makes this summary box look a bit less busy:
![image](https://user-images.githubusercontent.com/49409390/82906403-2fa96300-9f33-11ea-8227-f408e98064f0.png)